### PR TITLE
Properly display profile view for private users

### PR DESCRIPTION
**Pull request by jamietsao**
https://github.com/jamietsao/codepath-mapstagram/issues/116

I put `?` for the stat counts for now, otherwise everything gets pushed up and it looks funny. `-` could be another option.  Here's what the private profile view looks like now:

screen shot 2015-03-23 at 5 30 49 pm

### CODE REVIEW

File: Mapstagram/ViewControllers/ProfileViewController.swift [hunk 1]
```diff
@@ -35,6 +35,11 @@ class ProfileViewController: UIViewController {
             gridCollectionView.dataSource = self
         }
     }
+    @IBOutlet weak var privateProfileView: UIView! {
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ProfileViewController.swift]: Added a new UIView in the same spot as the tableView of media.  Defaults to hidden, but is shown if user is private

File: Mapstagram/ViewControllers/ProfileViewController.swift [hunk 2]
```diff
@@ -97,20 +102,30 @@ class ProfileViewController: UIViewController {
...
-        // load user media
-        MBProgressHUD.showHUDAddedTo(self.view, animated: true)
-        loadMedia {
-            MBProgressHUD.hideHUDForView(self.view, animated: true)
-            return
+        return false
+    }
+    
+    func updateUI() {
+        // need to get relationship status first before doing anything else
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ProfileViewController.swift]: In order to determine if user is private, we must retrieve relationship info first.  This means I had to move the other work (retrieving of user profile and user media) into a closure that is invoked on completion.

File: Mapstagram/ViewControllers/ProfileViewController.swift [hunk 3]
```diff
@@ -97,20 +102,30 @@ class ProfileViewController: UIViewController {
...
+        refreshRelationship() {
+            if self.isPrivateProfile() {
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/ProfileViewController.swift]: If private, refresh user profile data with `?` for stats and unhide UIView that shows "This user is private".

File: Mapstagram/ViewControllers/ProfileViewController.swift [hunk 4]
```diff
@@ -97,20 +102,30 @@ class ProfileViewController: UIViewController {
...
+                self.refreshUserInfoUI(countsPlaceholder: "?")
```
Comment (kristen) [inline Mapstagram/ViewControllers/ProfileViewController.swift]: Let's see what it looks like with the dash

**Comment by kristen** [comment]
Looks great. Merge it when you update with the dash

**Comment by jamietsao** [comment]
Updated to use dash

screen shot 2015-03-23 at 10 49 17 pm

**Pull request merged**